manager/logbroker/subscription.go

```diff
@@ -182,6 +183,11 @@ func (s *subscription) match() {
 				continue
 			}
 			for _, task := range tasks {
+				// if we're not following, don't add tasks that aren't ready yet
+				if !s.follow() && task.Status.State < api.TaskStateRunning {
+					s.errors = append(s.errors, errors.Errorf("task %v is not yet running", task.ID))
```

Comment:
I think doing that in no-follow mode was a bug that gets fixed as a side effect of this change. It wouldn't hurt to explicitly avoid adding those tasks as well, in no-follow mode.